setup.py

```diff
@@ -20,6 +20,10 @@
 from setuptools import setup, Extension, find_packages
 from setuptools.command.build_ext import build_ext
 
+backend = os.environ.get("BACKEND", "qubit")
+backend = backend.replace("lightning_", "")
+if backend not in ["kokkos", "qubit"]:
+    raise ValueError(f"Invalid backend {backend}.")
```

Comment:
Is breaking the device name needed?

Reply:
Something like `any(sub_str in string for string in str_list)` could work here instead.

Reply:
`setup.py` updated.